src/Kip.ts

```diff
@@ -29,6 +29,8 @@ function setup(config: RaptConfig): KalturaInteractiveVideo {
     const uiconfRaptData: any = uiconfData.rapt || {};
     const uiconfPlaybackData: any =
         (uiconfData.player && uiconfData.player.playback) || uiconfData.playback || {};
+    const uiconfEnvData: any = uiconfData.provider?.env?.serviceUrl || {};
```

Comment:
if `uiconfEnvData` represents only the serviceUrl from the uiconfData it's better to destruct it:
```
const { serviceUrl } = uiconfData.provider?.env;
```